### ISSUE TITLE
Training without validation set

### ISSUE THREAD

**Issue by dhkhey** [rrmenon10/ADAPET#35]
Hi,

Thanks for sharing this work.
I am wondering if there is a simple way to run this code without having a validation set?
Also, if I run ADAPET on generic dataset, do you recommend doing hyperparameter search? 

Thanks.

**Comment by rrmenon10** [comment]
In the config file, if you set the value for eval_every to be the same as the num_batches hyperparameter, the model will get evaluated just once at the end, and that model would get saved.

Regarding the hyperparameter search, we didn't actually tune many hyperparameters in our code for any dataset. Choosing the best model based on validation set evaluations at different points in training could be considered a hyperparameter (setting the eval_every value). But, in practice, we found that training for just 250 batches and evaluating the model at the end of that was enough to get good performance.

**Comment by rrmenon10** [comment]
Closing due to inactivity.
